src/pages/index.tsx

```diff
@@ -305,152 +320,41 @@ export default function Home() {
           <a className="text-lg">Why do you carry in your wallet?</a>
         </div>
         <div className="flex flex-row">
-          {whyDoYouCarryContent.map((content, i) => (
-            <div
-              className="flex flex-col space-x-36 py-14 items-center justify-center"
-              key={i}
-            >
+          {whyDoYouCarryContent.map((content, i) => {
+            const isActive = imagesToShow === content.text;
+            return (
               <div
-                className="flex items-center flex-col px-24 group cursor-pointer"
-                onClick={() => setImagesToShow(content.text)}
+                className="flex flex-col space-x-36 py-14 items-center justify-center"
+                key={i}
               >
-                {imagesToShow === content.text ? (
-                  <>
-                    <Image
-                      src={content.activeImage}
-                      width={96}
-                      height={71}
-                      alt="Mens Bags"
-                      style={{ height: "100%", width: "auto" }}
-                    />
-                    <div className="absolute group-hover:text-orange-600 text-2xl flex justify-center translate-y-32">
-                      <BsChevronCompactDown />
-                    </div>
-                  </>
-                ) : (
+                <div
+                  className="flex items-center flex-col px-24 group cursor-pointer"
+                  onClick={() => setImagesToShow(content.text)}
+                >
```

Comment:
Extract to new component, generally if you're mapping a bunch of JSX it's good idea to extract

Reply:
component extracted